src/applications/vaos/components/StatusAlert.jsx

```diff
@@ -3,6 +3,8 @@ import { useLocation } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
+import { format, differenceInBusinessDays, isValid } from 'date-fns';
```

Comment:
You're now the team expert on `date-fns`!

Reply:
It's simple to use and has good [documentation](https://date-fns.org/docs/Getting-Started).